applications/openshift/master/file_permissions_cni_conf_s390x/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,2 @@
+---
+default_result: NOT-APPLICABLE
```

Comment:
So we're still using the default assertion in https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/master/file_permissions_cni_conf/tests/ocp4/e2e.yml#L3

But the CI we're running reports this as a pass:

```
    helpers.go:810: Result - Name: e2e-cis-node-master-file-groupowner-cni-conf - Status: PASS - Severity: medium
    helpers.go:921: Rule file_groupowner_cni_conf matched expected result
```

Is that part of the fix for https://github.com/ComplianceAsCode/ocp4e2e/pull/38 ?

Reply:
no this is different, I think maybe we should use Pass instead ?

Reply:
Oh - I think I got my rules mixed up.. Nevermind.